tests/loss/test_swav_loss.py

```diff
@@ -24,10 +22,7 @@ def test__sinkhorn_gather_distributed_dist_not_available(
             SwaVLoss(sinkhorn_gather_distributed=True)
         mock_is_available.assert_called_once()
 
-
-class TestSwaVLossUnitTest(unittest.TestCase):
-    # Old tests in unittest style, please add new tests to TestSwavLoss using pytest.
-    def test_forward_pass(self):
+    def test_forward_pass(self) -> None:
```

Comment:
This should be a parametrised test instead of iterating with 2 for-loops. 
Same applies to the other tests.